### ISSUE TITLE
Email certificate link is not working

### ISSUE THREAD

**Issue by spatialthoughts** [kartoza/prj.app#1140]
I held a certification workshop recently and issued certificates. When I click the 'email participants with a link to their certificates' it says success, but participants are not receiving any email. 

Can you please look into this?
https://changelog.qgis.org/en/qgis/certifyingorganisation/spatial-thoughts/course/qgis_custimizing-qgis-with-python_2020-02-15-2020-02-15/

**Comment by timlinux** [comment]
@ann26 can you look at this please?

@spatialthoughts keep filing those issues if you hit problems - we will fix as quickly as we can. Thanks!

**Comment by spatialthoughts** [comment]
Update: It seems the messages are being delivered but they are detected as Spam. See attached screenshot. Also name, organization name fields are byte strings with a 'b' prefix that should be converted to a string for the email

![Screenshot_20200219-073201](https://user-images.githubusercontent.com/5227506/74806971-542ae900-530d-11ea-811c-1d143f7c10a2.png)

I didn't realize the certificates are sent from the issuing organization's email. It will be better and preferable for the certificate email to come from qgis.org email. That looks more professional and can also avoid emails going to spam.

**Comment by timlinux** [comment]
Yes this also relates to #1144 - the advertised domain should be consistent everywhere.

**Comment by nickbearman** [comment]
+1 
Also having this issue with certificates that I have sent out. The participants claimed not to have recieved any emails (even in SPAM).

**Comment by timlinux** [comment]
@sumandari can you look into this please?

**Comment by nickbearman** [comment]
Just FYI, I have had one more person confirm they haven't recieved the certificate email, and it is not in their spam.

**Comment by sumandari** [comment]
@nickbearman thank you for your report. Could you please inform the detail, which course the attendees haven't received the certificate email?
I have looked into the log error and haven't found any issue related email recently. Your detail information will help to identify the issue.

**Comment by nickbearman** [comment]
Sure @sumandari 
It is the Introduction to QGIS: Spatial Data and Spatial Analysis | April 20, 2021 under Geospatial Training Solutions. 
https://changelog.qgis.org/en/qgis/certifyingorganisation/geospatial-training-solutions/course/qgis_introduction-qgis-spatial-data-spatial-analysis_2021-04-20-2021-04-28/

Participants impacted: (all UK time)
Juan Manuel Moreno, sent certificate through online system 10:14 30th April, user confirmed not recieved on 5th May 09:45, including checking spam. 
Simeon Shtebunaev, sent certificate 4th May 07:50, user confirmed not recieved 10:58 same day, including checking spam. 
I can check if there are others if you need more info. 
Best wishes,
Nick.

**Comment by spatialthoughts** [comment]
I would like to report another instance of a participant not receiving the certificate email. 

Participant: Ellie Joo
Course: Advanced QGIS https://changelog.qgis.org/en/qgis/certifyingorganisation/spatial-thoughts/course/qgis_advanced-qgis-online_2021-05-29-2021-05-30/
sent certificate 30th May, 2021 and not received email including in spam folder.

Other participants from the same batch received the email (though many reported it going to spam).

**Comment by sumandari** [comment]
@nickbearman, @spatialthoughts  thank you for the detail and update. 
 
We apologize for the inconvenience. We're still working to resolve the issue.

**Comment by spatialthoughts** [comment]
Thanks. 

Closing the loop on this one, the participant was able to find the certificate email by searching with the exact subject line in the Spam folder 'Certificate from Advanced QGIS (Online) Course'. So the issue wasn't about email delivery, but the email going to Spam.

**Comment by nickbearman** [comment]
Thanks, I have asked both my students to check again for 
"Certificate from Introduction to QGIS: Spatial Data and Spatial Analysis Course"
and will let you know what they say.
